### DISCUSSION TITLE
Can't log in with email

### DISCUSSION THREAD

**Discussion by bbtools-ps** [vercel/community#6590] [Account Issue] [answered]
### Question

When trying to log in using email I tried to send a `POST` request to the `https://api.vercel.com/registration`. I received the email prompting me to verify the token and after clicking a button or a link below the button in the email message I got this error:

Unable to verify your email. Please close this window and try clicking the link from your email again. If the issue persists, please contact support.

Because I use Vercel CLI to manually deploy the website when I try to log in using `vercel login` command and enter my email I get this error message:

Unexpected error: request to https://api.vercel.com/registration?mode=login&teamId=team_2GYN2YimRp8uISfVOeAeYV9G failed, reason: unable to verify the first certificate

When doing a manual deployment with `vercel --prod` this is the error I get:

Error: An unexpected error occurred in deploy: FetchError: request to https://api.vercel.com/v9/projects/prj_eMhBPGM4145pWE9MA5tpKndpW31A?teamId=team_2GYN2YimRp8uISfVOeAeYV9G failed, reason: unable to verify the first certificate

Everything has worked before but as of today, it stopped.

**Comment by amyegan** [comment]
Are you able to login to [vercel.com/login](https://vercel.com/login) via the web UI?

**Reply by bbtools-ps** [reply]
Yes. I can log in and see the project there. I just have a problem with the CLI tool which I use to make those manual deployments because the repository is on a private server.

**Reply by bbtools-ps** [reply]
@amyegan did you find anything?

**Reply by bbtools-ps** [reply, marked as answer]
@amyegan As I found out this has to do something with the firewall that I have at the company. I tested it out on my private PC and it worked well (as before).

**Reply by DaraBoth** [reply]
How to switch that to deploy with github repo everytime. I push new code? Please help me.

**Reply by pawlean** [reply]
@DaraBoth Can you start a new discussion as your request is not related to this thread? Thank you! 🙏🏼

[a comment by an automated account is not shown]
